test-apps/presentation-test-app/src/backend/web/BackendServer.ts

```diff
@@ -18,11 +18,12 @@ export default async function initialize(rpcInterfaces: RpcInterfaceDefinition[]
   await IModelHost.startup();
 
   // tell BentleyCloudRpcManager which RPC interfaces to handle
-  const rpcConfig = BentleyCloudRpcManager.initializeImpl({ info: { title: "presentation-test-app", version: "v1.0" } }, rpcInterfaces);
+  const paramsHolder = BentleyCloudRpcParams.wrap({ info: { title: "presentation-test-app", version: "v1.0" } });
```

Comment:
`BentleyCloudRpcParams` is `@internal` at the moment - shouldn't it be changed to `@public` if you're using it in test apps and example code?

Reply:
Making it public would force most of the internal parts of the RPC system to become public also. We are trying to deprecate the RPC system right now. Making more and more of it public would push that goal further out of reach.

I think my approach (BentleyCloudRpcParamsHolder) is the right way to bridge us to the future, here, especially since BentleyCloudRpc-anything is in fact an internal concept. The next RPC/REST/whatever system should have a more-generic way for apps to connect, that doesn't assume they are using Bentley's cloud infrastructure.

Reply:
I understand the goal, but deprecation should generally by done by first introducing the new approach, then deprecating the old one. But let's ignore that.

At the moment RPC system is partly `@public @deprecated` and partly `@internal`. If we have to choose between these two evils, I believe `@internal` is worse, as it hides the API from the docs. Maybe, at least, the APIs we expect consumers to call could be made `@public @deprecated` rather than `@internal`? I'm specifically referring to `BentleyCloudRpcParams` here. If some of it's attributes should be kept internal, you can tag them `@internal` individually, but I think `info` and `uriPrefix` could be made public, couldn't they?

Reply:
I see your point about info and uriPrefix. Let me try to come up with something for those…

Reply:
Thanks, I think with your latest changes it's much better.